_data/_2023_members.yml

```diff
@@ -18,3 +18,8 @@
   image_url: https://devagrawal09.github.io/portfolio/img/dp.jpg
   github: devagrawal09
   website: https://devagrawal09.github.io/portfolio/
+
+
+- name: Mahesh Gowda
+  short_bio: Hello! I'm a first year IT student with considerabl eexperience and motivation. I really like to experiment with APIs and open data.
```

Comment:
s/considerabl/considerable

Reply:
I do not at this time, although I will probably work on these and add them later.